README.md

```diff
@@ -1,33 +1,86 @@
 # HonzaBotner
 
-Bot for unoffici
+Bot for unofficial FIT CTU discord server.
 
 ## Configuration
 
-[`secrets.json`][secrets]:
+For configuration we use combination of standart appsettings.json file and secrets.json.
+Non-senesitive informations are stored in appsettings.json file and sensitive informations are stored in place outside of git reposiotry
+(for more details see [documentation][secrets])
+
+### App settings
+
+We use four different appsetings files.
+- **appsetings.json**: common configuration for all instances.
+- **appsetings.BotDev.json**: configuration for test server discord
+- **appsetings.CvutFit.json**: configuration for main FIT CTU discord server
+- **appsetings.Development.json**: configuration for local development.
 
+### Secrets
+
+Example file structure:
+
+[`secrets.json`][secrets]:
 ```json
 {
-  "CVUT:ClientId": "test",
-  "CVUT:ClientSecret": "test",
-  "DATABASE_URL": "Host=localhost;Database=HonzaBotner;Username=honza-bot;Password=postgres",
-  "Discord:Token": "test"
+    "CVUT:ClientId": "<client id>",
+    "CVUT:ClientSecret": "<client secret>",
+    "DATABASE_URL": "Host=localhost;Database=HonzaBotner;Username=honza-bot;Password=postgres",
+    "Discord:Token": "<discord bot token>"
 }
 ```
 
+CVUT id and secret can be found on [auth manager][oauth]. In case you don't have project there, you will need to create one
+and then create app in it (Web application). This projects also need to have access to following services:
 
+- urn:ctu:oauth:umapi.read
+- cvut:umapi:read
+
+Discord token can be found on [Discord developer portal][discordDev].
+
+Databse url in example is working with provided docker image which can be found in [compose file][compose]
+
+## Setup
+
+Before you can start working on bot you need to download latest released version of .NET. Availible version can be found [here][dotnet].
+We also need to have access to PostgresDB. For development we use docker image, so you will need to download it too, or you can use other
+Postgress instance.
+
+## Startip
+
+Before starting the bot itself we will need to start DB by following command:
+
+```sh
+docker-compose up -d
 ```
-dotnet tool install --global dotnet-ef
-cd src/HonzaBotner
-dotnet ef database update
-```
+if everything went well you will see `Starting postgres-botner ... done`.
+
+After that you can stratup the project in your favorite IDE or by typing `dotnet run --project .\src\HonzaBotner\`.
 
+## Migrations
 
+To modify databse scheme we need to add migration. For that we need to install tool whcih does that. This can be done by following command:
```

Comment:
```suggestion
To modify the database scheme we need to add migration. For that, we need to install a tool that does that. This can be done by the following command:
```